core/src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -1915,8 +1915,7 @@ class WorkspaceService(protected val userInfo: UserInfo, val dataSource: SlickDa
     )
 
     // todo: update this line as part of https://broadworkbench.atlassian.net/browse/CA-1220
-    policyGroupsToRoles.toList.traverse {case (email, roles) =>
-      googleIamDao.addIamRoles(GoogleProject(googleProject.value), email, MemberType.Group, roles)} // addIamRoles logic includes retries
```

Comment:
I would add a comment here about these calls intentionally being called sequentially rather than in parallel and why we are doing that.  CA-1220 is more about reducing the # of times we call Google, which is also good, but is addressing a slightly different problem.  The goal here is that if anybody needs to modify this in the future, they are aware of why exactly we chose this very specific implementation over the more obvious/efficient choice to run in parallel.